docs/7.x/cluster.md

```diff
@@ -410,6 +410,15 @@ drwxr-xr-x. 5 user user 4.0K Jan 14 09:08 packages
 You can execute the `./upgrade` script to upload and upgrade in one go,
 or you can upload the update and then execute `gravity upgrade` command.
 
+!!! note "Disabling existing functionality when upgrading"
+`gravity upgrade` will not uninstall existing applications that have been disabled in the upgrade manifest.
+Example:
+    We have a cluster that has the monitoring app enabled.
```

Comment:
Are you sure the indented docs will render correctly? Atleast in github markdown it doesn't seem to render, I'm not sure about our docs engine.

Reply:
the note in the markdown renders the same as the other notes in the doc. I have followed the example of the other notes.